pyfr/integrators/dual/phys/controllers.py

```diff
@@ -46,6 +46,14 @@ def advance_to(self, t):
         if t < self.tcurr:
             raise ValueError('Advance time is in the past')
 
+        scoeffs = [self.stepper_coeffs(s, self._dt)
+                   for s in range(self.nstages)]
+
         while self.tcurr < t:
-            self.pseudointegrator.pseudo_advance(self.tcurr)
+            for s, (sc, tc) in enumerate(zip(scoeffs, self.c)):
```

Comment:
Is there any scope to trying to move this loop to the physical stepper?  As it is not really a controller related thing.